.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 sudo: required
-dist: trusty
```

Comment:
Does this need to be removed?  or was it unnecessary to begin with?

Reply:
Travis was erroring due to the ongoing numpy issue. This was an attempt to resolve by making travis.yml more like party service  that was not having these issues